src/picomsg/src/oper_db_writer.h

```diff
@@ -11,8 +12,13 @@ extern "C" {
 bool oper_db_writer_terminator_state_change(const uint32_t terminator_id, const enum pm_terminator_conn_state terminator_state);
 
 bool oper_db_writer_set_terminator_tunnel_counters(const uint32_t terminator_id, const uint64_t count);
-bool oper_db_writer_set_terminator_vrf_tunnel_counter(const uint32_t terminator_id, const char *tenant_id, 
+bool oper_db_writer_set_terminator_vrf_tunnel_counter(const uint32_t terminator_id, const char *tenant_id,
                                                 const char *account_id, const char *vrf_name, const uint64_t count);
+bool oper_db_writer_del_terminator_vrf_tunnel_counter(const uint32_t terminator_id, const char *tenant_id,
+                                                const char *account_id, const char *vrf_name);
+bool oper_db_writer_set_terminator_tunnel_oper_data(const struct terminator_tunnel_oper_data *oper_data, bool write_uptime);
+bool oper_db_writer_remove_terminator_tunnel_oper_data(const char *vrf_name, const char *device_id);
```

Comment:
Use consistent names - If you use the term "del" to denote deleting from oper (as in `oper_db_writer_del_terminator_vrf_tunnel_counter()`), then also use it here (`remove` -> `del`)